app/javascript/App.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Greeting from "./components/Greeting.js";
+
+const App = () => {
+  return (
+    <main>
+      <h1>Hello Everyone</h1>
+      <Greeting />
+    </main>
```

Comment:
- [ ] Please ensure there is a route component that leads to the Greeting component. This will ensure there is proper client-side navigation